config/jobs/build-drivers/build-new-amazonlinux.yaml

```diff
@@ -86,13 +86,46 @@ postsubmits:
             memory: 6Gi
       nodeSelector:
         Archtype: "x86"
+  - name: build-new-drivers-amazonlinux-postsubmit-arm
+    decorate: true
+    skip_report: false
+    agent: kubernetes
+    branches:
+      - ^master$
+    run_if_changed: '^driverkit/config/[a-z0-9.+-]{5,}/aarch64/amazonlinux_*'
+    spec:
+      serviceAccountName: driver-kit
+      containers:
+      - command:
+        - /workspace/build-drivers.sh
```

Comment:
I think we should also pass an additional env to `build-drivers.sh` to set the TARGET_ARCH (otherwise we end up trying to build all drivers on each architecture).

Note that `build-drivers.sh` already has a `TARGET_ARCH` (that is 4th argument passed); i think we could just default it to output of `uname -m` (ie: architecture we are currently running on), when it is not passed from cmdline; this will help us avoid touching all config jobs once again!
Basically, we should update this line: https://github.com/falcosecurity/test-infra/blob/86fc92a0816cf53c474fe0cf4801543554a2b4c7/images/build-drivers/build-drivers.sh#L21 adding the default to `$(uname -m)`.

@maxgio92 do you agree? I think it might be a good behavior to default to current architecture.

Reply:
@FedeDP got it. As of now by default, the wrapper script runs the DBG for all architectures (i.e. `$4` is empty), and I think it should remain as so.
Instead, I would explicitly pass the architecture as argument, as already done with the Linux distrubution (i.e. `amazonlinux`). As the argument accepted as of now is onyl positional, I would introduce the setting of the default value from the environment.
WDYT?

Reply:
While that would also work, i think that we generally (so, by default) might want to only build drivers for the platform we are currently running on, right?
I mean, it seems a sensible default! But i haven't got a strong opinion about this :) fully open to discuss!

Reply:
It's not a big deal, we can go on with this PR @FedeDP

Reply:
Yep the main concern here is that defaulting to "build configs for any architecture" it's gonna work only on x86 nodes, where qemu is supported.
Imo it makes sense to instead default to "build just your own arch", but again either way is fine ahha

Reply:
Btw i'd give this a try so that next time kernel-crawler runs we will test native arm64 builds.
We can then smooth the rough edges ;)